### ISSUE TITLE
--ramdb fails to write back to disk if database is too large

### ISSUE THREAD

**Issue by hydrrrrr** [RhetTbull/osxphotos#1203]
** Before submitting a bug report, please ensure you are running the most recent version of osxphotos and that the bug is reproducible on the latest version **

pipx upgrade osxphotos

**Describe the bug**
osxphoto run the export 100% then crashes. 

**To Reproduce**
./osxphotos export /Volumes/Memories/export/ --directory "{folder_album}" --touch-file --exiftool --ramdb --keyword-template "{label}"

Was exporting, and it said it crashed after 100%. Told me to upload crashfile.

**Expected behavior**
Not crash at the end. 

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. which version of macOS] Latest MacOS. 13.5.1
 - osxphotos version 0.63.0 - Python 3.11.2, macos 13.5.1, arm64.
 -
 - 
**Additional context**
[osxphotos_crash.log](https://github.com/RhetTbull/osxphotos/files/12597374/osxphotos_crash.log)

**Comment by RhetTbull** [comment]
In the future, please fill out the bug template as its purpose is to provide helpful context for finding and fixing the issue.  In this case, I'm able to see the issue from the crash dump. You'll need to run without `--ramdb` until I can get this fixed.

**Comment by RhetTbull** [comment]
```
Traceback (most recent call last):
  File "tenacity/__init__.py", line 382, in __call__
  File "osxphotos/export_db.py", line 904, in write_to_disk
sqlite3.DataError: query string is too large
```

The issue is use of execute script to write the database. Instead, this should use `Connection.backup()`:

```python
conn_on_disk.cursor().executescript(dbdump.read())
```

**Comment by hydrrrrr** [comment]
> In the future, please fill out the bug template as its purpose is to provide helpful context for finding and fixing the issue. In this case, I'm able to see the issue from the crash dump. You'll need to run without `--ramdb` until I can get this fixed.

Sorry about that updated now. I will do.  Absolutely amazing to be able to use osxphotos. Thank you so much.

**Comment by RhetTbull** [comment]
@hydrrrrr I think this is fixed in [v0.63.1](https://github.com/RhetTbull/osxphotos/releases/tag/v0.63.1). Unfortunately due to the nature of the bug your export database won't have been written from ram back to disk due to the crash so you'll lose export state. You could re-run with `--update --cleanup` which would re-export some files then cleanup any left over files. I'd do this twice then drop the `--cleanup` if you don't want to continue use that.

**Comment by RhetTbull** [comment]
@all-contributors please add @hydrrrrr  for bug
